var/spack/repos/builtin/packages/libsonatareport/package.py

```diff
@@ -0,0 +1,44 @@
+# Copyright 2013-2018 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class Libsonatareport(CMakePackage):
```

Comment:
As discussed on Slack, we should remove old package and make this as libsonata-report